# HG changeset patch
# Parent c88f8e8921fb6399cf95b5899d2acc60951dddd1
Bug 665934 - fix bug: arev was meant instead of amap - r=upstream

This patch has been taken in ANGLE upstream as r699.

diff --git a/gfx/angle/src/compiler/preprocessor/atom.c b/gfx/angle/src/compiler/preprocessor/atom.c
--- a/gfx/angle/src/compiler/preprocessor/atom.c
+++ b/gfx/angle/src/compiler/preprocessor/atom.c
@@ -330,17 +330,17 @@ static int GrowAtomTable(AtomTable *atab
             newrev = malloc(sizeof(int)*size);
             atable->size = 0;
         }
         if (!newmap || !newrev) {
             /* failed to grow -- error */
             if (newmap)
                 atable->amap = newmap;
             if (newrev)
-                atable->amap = newrev;
+                atable->arev = newrev;
             return -1;
         }
         memset(&newmap[atable->size], 0, (size - atable->size) * sizeof(int));
         memset(&newrev[atable->size], 0, (size - atable->size) * sizeof(int));
         atable->amap = newmap;
         atable->arev = newrev;
         atable->size = size;
     }
